aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -393,6 +403,20 @@ module aptos_framework::account {
         (signer, signer_cap)
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    /// Coin management methods.
+    ///////////////////////////////////////////////////////////////////////////
+
+    public(friend) fun register_coin<CoinType>(account_addr: address) acquires Account {
+        let account = borrow_global_mut<Account>(account_addr);
+        event::emit_event<CoinRegisterEvent>(
+            &mut account.coin_register_events,
+            CoinRegisterEvent {
+                type_info: type_info::type_of<CoinType>(),
+            },
+        );
+    }
+
```

Comment:
Interesting, are more things going to be on the account?

Reply:
yes, the intention is to see if we can't get AptosCoin there as well... that will return p2p transactions to be the same cost as before fine grained storage.